python/setup.cfg

```diff
@@ -26,3 +26,6 @@ docstring-convention = google
 
 [aliases]
 test = pytest
+
+[isort]
+profile=black
```

Comment:
took me a while to find this... next time maybe do the setup in 1 commit and apply the format in another :sweat_smile:

Reply:
Forgot this is github not gerrit 🤣